apps/admin-portal/src/components/index.ts

```diff
@@ -1,5 +1,5 @@
 /**
- * Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
This file was added last year I think we don't need to change the year here.

Reply:
Fixed with comment `48590b7`